cryptoapi_windows.go

```diff
@@ -9,16 +9,66 @@ import (
 	"time"
 
 	"golang.org/x/sys/windows/registry"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var (
+	cryptoApiFlagGroup            = cflag.NewGroup(flagGroup, "capi")
+	cryptoApiFlagLogicalStoreName = cflag.String(cryptoApiFlagGroup, "store", "Root",
```

Comment:
The flag `store` to indicate logical store is a bit ambiguous since logical and physical stores are both stores.  Maybe change this to `logical-store`?  (Also maybe rename the flag `scope` below to `physical-store` for consistency.)